pkg/controller/sparkapplication/job.go

```diff
@@ -30,85 +36,121 @@ import (
 	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/config"
 )
 
-type submissionJobManager struct {
+const (
+	sparkSubmitPodMemory = "256Mi"
```

Comment:
I think we can use a lower memory request.

Reply:
lowered request cpu/memory to 100m/100Mi  . limits are 250m/256Mi